### PR TITLE
Fix typo, grammar, punctuations in NATS docs

### LINKED ISSUE #770: Check NATS docs

### PR DESCRIPTION
**Pull request by kumaranvpl**
# Description

Fix typo, grammar, punctuations in NATS docs

Fixes #770 

## Type of change

- [x] Documentation (typos, code examples, or any documentation updates)

## Checklist

- [ ] My code adheres to the style guidelines of this project (`scripts/lint.sh` shows no errors)
- [x] I have conducted a self-review of my own code
- [x] I have made the necessary changes to the documentation
- [x] My changes do not generate any new warnings
- [x] Both new and existing unit tests pass successfully on my local environment by running `scripts/test-cov.sh`
- [ ] I have ensured that static analysis tests are passing by running `scripts/static-anaylysis.sh`

### CODE REVIEW

File: docs/docs/SUMMARY.md [hunk 1]
```diff
@@ -37,7 +37,7 @@
     - [Logging](getting-started/logging.md)
     - [Config Management](getting-started/config/index.md)
 - [Kafka](kafka/index.md)
-    - [Subscription](kafka/Subscriber/index.md)
+    - [Subscribing](kafka/Subscriber/index.md)
```
Comment (Lancetnik) [inline docs/docs/SUMMARY.md:40]: We should use the same section naming at all *RMQ*, *Kafka*, *NATS* and *Tutorial* blocks

**Review by Lancetnik — CHANGES_REQUESTED** [review]
Can you also take a look at logging page section too, please?

**Comment by Lancetnik** [comment]
Also, can you check that all external links in the NATS section are valid, please?

File: docs/docs/SUMMARY.md (hunk 1 above)
Comment (kumaranvpl) [reply docs/docs/SUMMARY.md:40]: Someone messed it up - https://github.com/airtai/faststream/blame/0.2.0-NATS/docs/docs/summary_template.txt#L40. Let me fix it.

Comment (kumaranvpl) [reply docs/docs/SUMMARY.md]: Done.

**Comment by kumaranvpl** [comment]
> Can you also take a look at logging page section too, please?

This one - https://faststream.airt.ai/0.1/getting-started/logging/ ?

**Comment by Lancetnik** [comment]
Yep

**Comment by kumaranvpl** [comment]
> Also, can you check that all external links in the NATS section are valid, please?

Done. All the links works. 
There was a broken link which should have pointed to NATS youtube channel. I fixed it while going through NATS docs.

**Comment by Lancetnik** [comment]
Thanks, I just forgot about it, sorry

**Comment by Lancetnik** [comment]
You can take a look at all `main` and `NATS` branch docs differences here: https://github.com/airtai/faststream/pull/692/files#diff-0f4e057b715abd5c87898813ecca5eff9c92c905f94e93e1cb483d98fdbba34c

**Comment by kumaranvpl** [comment]
> Yep, I added stuctlog example to it

Went through whole logging section again and fixed issues. Please check.

File: .faststream_gen/getting-started/logging.md [hunk 2]
```diff
@@ -111,7 +111,7 @@ from faststream import context
 log_context: dict[str, str] = context.get_local("log_context")
 ```
 
-This way, all broker hanlers able to get access to you broker right from the context:
+This way, all broker handlers cant get access to your broker right from the context:
```
[the remark this answers is not in the document]
Comment (kumaranvpl) [reply .faststream_gen/getting-started/logging.md:114]: Ahhh. Sorry. Typing mistake.

**Comment by Lancetnik** [comment]
Well, looks like the job is done, thanks!
Probably, I should spend all this project donations on english lessons...

**Review by Lancetnik — APPROVED** [review]

**Comment by Lancetnik** [comment]
I have to go, sorry
Davor or Tvrtko can merge this branch, I already checked logging & NATS docs, so there should no any problems

**Comment by kumaranvpl** [comment]
> > You can take a look at all `main` and `NATS` branch docs differences here: https://github.com/airtai/faststream/pull/692/files#diff-0f4e057b715abd5c87898813ecca5eff9c92c905f94e93e1cb483d98fdbba34c
> 
> I am going through all the docs changes now. Will let you know once I am done.

Rest of the docs changes looks fine. No significant changes. We can merge this.

**Review by davorrunje — APPROVED** [review]

**Pull request merged**